lib/pyfrc/test_support/pytest_plugin.py

```
@@ -59,6 +59,9 @@ def pytest_runtest_setup(self):
                        "If your robot class has an __init__ function, it must call super().__init__()!"
     
     def pytest_runtest_teardown(self, nextitem):
+        # Let any child threads run in realtime to allow cancelling if it
+        # has been implemented.
+        self._fake_time.freerun_children()
```

And if it hasn't been implemented? Maybe throw an exception the next time they try to delay?